federatedscope/core/auxiliaries/trainer_builder.py

```diff
@@ -122,7 +126,11 @@ def get_trainer(model=None,
 
     # fed algorithm plug-in
     if config.fedprox.use:
-        from federatedscope.core.trainers.trainer_fedprox import wrap_fedprox_trainer
-        trainer = wrap_fedprox_trainer(trainer)
+        if config.federate.method.lower() == "fedprox":
```

Comment:
We should consider the scenarios that `fedprox.use=True` but `federate.method='fedem'/'ditto'/'fedbn'/...`